src/Microsoft.AspNet.OData.Shared/EdmODataAPIHandlerFactory.cs

```diff
@@ -0,0 +1,49 @@
+﻿//-----------------------------------------------------------------------------
+// <copyright file="EdmODataAPIHandlerFactory.cs" company=".NET Foundation">
+//      Copyright (c) .NET Foundation and Contributors. All rights reserved. 
+//      See License.txt in the project root for license information.
+// </copyright>
+//------------------------------------------------------------------------------
+
+using System;
+using Microsoft.OData.Edm;
+using Microsoft.OData.UriParser;
+
+namespace Microsoft.AspNet.OData
+{
+    /// <summary>
+    /// Factory class for OData API Handlers for typeless entities.
+    /// </summary>
+    internal abstract class EdmODataAPIHandlerFactory
+    {
+        protected EdmODataAPIHandlerFactory(IEdmModel model)
+        {
+            Model = model;
+        }
+
+        /// <summary>
+        /// The IEdmModel for the factory.
+        /// </summary>
+        public IEdmModel Model { get; private set; }
```

Comment:
nit: Given that this property is never set from other methods of the class (other than constructor of course), you could have left out `private set;` altogether